docs/additional-configuration.adoc

```diff
@@ -215,6 +215,16 @@ kubectl create secret -n "$NAMESPACE" generic git-ssh-key \
   --from-file=ssh_config=/tmp/ssh_config \
   --from-literal=passphrase="$PASSPHRASE"
 ----
++
+Note: If a passphrase is provided, the DevWorkspace operator adds a postStart event that starts the SSH agent and adds the passphrase.
+The DevWorkspace operator also modifies the bashrc file to configure the `SSH_AGENT_PID` and `SSH_AUTH_SOCK` environment variables.
+If you are automatically mounting your own bashrc file with a ConfigMap (see link:additional-configuration.adoc#automatically-mounting-volumes-configmaps-and-secrets[Automatically mounting volumes, configmaps, and secrets])
+you must add the following in your bashrc file:
++
+[source,bash]
+----
+[ -f /home/user/ssh-environment ] && source /home/user/ssh-environment
```

Comment:
In the [implementation](https://github.com/devfile/devworkspace-operator/pull/1307/files#diff-12b01779c7cf67e18ef8d7bf46eb788bb6b640db0fa051804f2ef7b31a50d4d5R22) of the postStart event, $HOME is used instead of `/home/user/` as the workspace may be using a different image than the UDI, which may have a different value for `$HOME`.

So maybe this should instead be:
`[ -f $HOME/ssh-environment ] && source $HOME/ssh-environment`

Reply:
Fixed